lib/models/os.coffee

```diff
@@ -330,13 +343,25 @@ getOsModel = (deps, opts) ->
 	# 	fs.writeFile('foo/bar/config.json', JSON.stringify(config));
 	# });
 	###
-	exports.getConfig = (nameOrId, callback) ->
+	exports.getConfig = (nameOrId, options = {}, callback) ->
+		defaultOpts =
+			appUpdatePollInterval: 1
```

Comment:
I know this defaults to 1 in the UI, but are we sure it's in minutes at the API level? `resin-device-config` [defaults](https://github.com/resin-io-modules/resin-device-config/blob/master/lib/config.coffee#L77) to 60000... Feels like somebody must be wrong there.